crates/sui-sdk/README.md

```diff
@@ -1,6 +1,138 @@
-# Rust SDK examples
+# Sui Rust SDK
 
-This directory contains examples of interacting with a Move language smart contract using the Sui Rust SDK. See the [introduction to the Rust SDK](https://github.com/MystenLabs/sui/blob/main/doc/src/build/rust-sdk.md) for additional details.
+This crate provides the Sui Rust SDK. For the crate documentation, see [https://docs.rs/sui_sdk/](https://docs.rs/sui_sdk/)
+
+## Getting Started
```

Comment:
```suggestion
## Getting started
```
Title is title case, but following headings are sentence case.

Reply:
Fixed